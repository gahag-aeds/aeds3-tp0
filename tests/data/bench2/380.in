380 190
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 
Add 203 328
Max 102 227
Sub 88 213
Min 159 284
Sub 118 243
Min 161 286
Sub 116 241
Max 69 194
Add 199 324
Min 124 249
Add 7 132
Min 206 331
Sub 186 311
Sum 225 350
Add 39 164
Max 17 142
Add 209 334
Min 204 329
Add 15 140
Min 159 284
Add 181 306
Max 55 180
Sub 64 189
Sum 221 346
Add 27 152
Max 109 234
Sub 134 259
Sum 218 343
Sub 32 157
Max 3 128
Sub 116 241
Min 226 351
Sub 96 221
Max 204 329
Sub 122 247
Sum 213 338
Sub 102 227
Min 238 363
Add 19 144
Sum 38 163
Add 107 232
Max 25 150
Sub 236 361
Max 30 155
Add 241 366
Min 20 145
Add 47 172
Max 187 312
Add 215 340
Max 53 178
Add 83 208
Min 133 258
Add 107 232
Max 139 264
Add 99 224
Min 125 250
Add 247 372
Min 224 349
Sub 80 205
Sum 16 141
Sub 218 343
Sum 196 321
Add 241 366
Sum 59 184
Add 137 262
Max 109 234
Add 9 134
Min 238 363
Sub 84 209
Min 27 152
Add 13 138
Sum 190 315
Add 43 168
Max 248 373
Sub 212 337
Max 21 146
Add 13 138
Sum 4 129
Add 199 324
Max 220 345
Sub 48 173
Min 28 153
Sub 98 223
Max 146 271
Sub 166 291
Sum 189 314
Sub 16 141
Max 150 275
Sub 158 283
Max 95 220
Sub 158 283
Sum 122 247
Sub 28 153
Max 144 269
Sub 172 297
Sum 164 289
Sub 244 369
Sum 181 306
Add 147 272
Sum 73 198
Add 199 324
Min 160 285
Add 1 126
Max 234 359
Add 145 270
Sum 212 337
Sub 246 371
Max 158 283
Add 207 332
Max 191 316
Add 115 240
Min 254 379
Sub 218 343
Sum 204 329
Add 145 270
Min 121 246
Sub 138 263
Min 152 277
Sub 8 133
Max 42 167
Sub 238 363
Max 165 290
Add 155 280
Max 12 137
Add 55 180
Min 72 197
Add 175 300
Max 36 161
Sub 244 369
Max 60 185
Add 101 226
Sum 181 306
Add 211 336
Sum 101 226
Sub 152 277
Max 101 226
Sub 50 175
Max 143 268
Sub 250 375
Max 2 127
Add 79 204
Max 102 227
Add 1 126
Max 34 159
Sub 52 177
Min 137 262
Sub 154 279
Sum 189 314
Sub 34 159
Max 162 287
Sub 222 347
Sum 17 142
Sub 72 197
Sum 122 247
Sub 20 145
Min 118 243
Sub 186 311
Min 187 312
Sub 146 271
Sum 167 292
Sub 246 371
Sum 246 371
Add 85 210
Max 194 319
Sub 84 209
Min 236 361
Sub 32 157
Max 133 258
Add 125 250
Sum 20 145
Sub 126 251
Sum 195 320
Add 121 246
Max 126 251
Add 229 354
Max 172 297
Sub 254 379
Sum 128 253
Add 107 232
Sum 33 158
Add 27 152
Sum 74 199
Add 41 166
Sum 91 216
Sub 188 313
Max 61 186
Sub 200 325
Sum 111 236
Add 247 372
Min 91 216
