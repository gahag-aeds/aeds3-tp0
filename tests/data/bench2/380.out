99
-1
-2
99
-3
1
6390
101
2
-3
102
6558
97
6531
101
0
102
6467
0
5843
102
102
2
100
103
-7
101
-5
1
6329
6657
5991
97
2
3
6903
104
106
6812
104
5
103
6797
102
99
5741
100
6148
6144
5976
-10
104
6570
100
101
0
6442
-7
-8
104
99
104
-6
105
105
6585
6235
99
100
104
99
106
-4
6679
98
6433
5997
-6
-3
6021
5924
95
-5
95
6049
6171
96
96
6123
5925
6050
6150
103
6006
-5
